packages/next-tinacms-cloudinary/package.json

```diff
@@ -7,6 +7,15 @@
   ],
   "typings": "dist/index.d.ts",
   "license": "Apache-2.0",
+  "buildConfig": {
+    "entryPoints": [
+      "src/index.ts",
+      {
+        "name": "src/handlers.ts",
+        "target": "node"
+      }
```

Comment:
We can target different environments from the same package, by default the entrypoint is `src/index.ts`, if you specifiy `buildConfig.entryPoints`, you'll be in control. If you only give a string, we'll assume `browser` as the target.